### PR TITLE
Contribution day organizer handbook

### PR DESCRIPTION
**Pull request by gabrieldagama**
## Purpose of this pull request

This pull request (PR) imports the https://opensource.magento.com/contribution-day-handbook#contribution-backlog to dev docs.

## Affected DevDocs pages

-  /community/resources/contribution-day-organizer-handbook.html

### CODE REVIEW

File: src/community/resources/contribution-day-organizer-handbook.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+| 9AM-10AM  | Coffee and Introductions                                                          | Start the event off with an introduction round for the attendees, a description of the venue, Wi-Fi information, social media information (hashtags), communication mediums (i.e. Slack channel), highlight people to contact for help and key locations such as toilets, coffee and smoking areas. This helps to make sure people feel as comfortable as possible at the event. |
+| 10AM-11AM | How to contribute, find issues to work on and set up your development environment | An introduction on how to go about contributing, how to find issue to work on and how to get a local dev environment going. This is all about helping give attendees context which is especially important for first-time contributors. Unless your attendees are all veteran Magento contributors, consider having some amount of time allotted for this.                       |
```
Comment (BarnyShergold) [inline src/community/resources/contribution-day-organizer-handbook.md:123]:
```suggestion
| 10AM-11AM | How to contribute, find issues to work on and set up your development environment | An introduction on how to go about contributing, how to find issues to work on and how to get a local development environment going. This is all about helping give attendees context, which is especially important for first-time contributors. Unless your attendees are all veteran Magento contributors, consider having some amount of time allotted for this.                       |
```

File: src/community/resources/contribution-day-organizer-handbook.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+| 11AM-1PM  | Development Time                                                                  | Get hacking!                                                                                                                                                                                                                                                                                                                                                                     |
+| 1PM-2PM   | Lunch                                                                             | nom nom                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (BarnyShergold) [inline src/community/resources/contribution-day-organizer-handbook.md:125]: Choose a better phrase "nom" is slang and this should be avoided as English as a Second Language readers may not understand

File: src/community/resources/contribution-day-organizer-handbook.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+| 5 mins    | Thank you and Wrap Up                                                             | A final note from the main organizer to wrap up the event: thank everyone for attending and remind everyone to keep an eye on their inbox for a [feedback survey](#feedback).                                                                                                                                                                                                                 |
+
+### Tracking Contributions
+The Magento Community Engineering team would love it if you are able to track activity and contributions that arise out of your event! This helps us highlight the benefits for the community, from the community, to the Magento ecosystem and provides us with evidence for further supporting such community events in the future. We are particularly interested in understanding how a contributor's activity evolves before, during and after a Contribution Day event, so information we are most interested in is:
+
+*  Number of participants
+
+*  GitHub.com usernames of contributors
```
Comment (BarnyShergold) [inline src/community/resources/contribution-day-organizer-handbook.md:135]: Attendees MUST agree to this sharing of information

File: src/community/resources/contribution-day-organizer-handbook.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+*  URLs to pull requests and issues filed during the event
+
+## After the Event
+Once the event is completed it is important to gather feedback from the attendees and to publish results and a review. This will help to build a stronger connection between the attendees and the event itself and it will also help you, as the organizer, to find out areas of improvement for the next event.
+
+We would also love to keep track of how many contributions were created as part of your event! Keeping track of pull requests issued as part of the event would be greatly appreciate on our part.
```
Comment (BarnyShergold) [inline src/community/resources/contribution-day-organizer-handbook.md:142]:
```suggestion
We would also love to keep track of how many contributions were created as part of your event! Keeping track of pull requests issued as part of the event would be greatly appreciated on our part.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
A great addition to the devdocs. Please note the suggestions for additional information AND GDPR issues.

Please remove all the blank lines between items in lists as this is causing Linting errors - check for others.

It's definitely a page that should be added to over time!

File: src/community/resources/contribution-day-organizer-handbook.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+*  GitHub.com username
+
+*  Community status: First time contributor? Part of the Magento GitHub Community Maintainer team?
+
+*  Areas of expertise? (Frontend, backend, QA, Docs)
+
+*  Project or area they are interested in contributing to?
+
+Having the above information can also be useful as feedback into [issue backlog preparation](#contribution-backlog) as well as helping to shape the goal and/or agenda for the event.
```
[the remark this answers is not in the document]
Comment (gabrieldagama) [reply src/community/resources/contribution-day-organizer-handbook.md:85]: I guess that depends on the region, but we can mention about data regulations. Good point.

File: src/community/resources/contribution-day-organizer-handbook.md (hunk 3 above)
Comment (gabrieldagama) [reply src/community/resources/contribution-day-organizer-handbook.md:135]: Agree, it is probably better to not have such information, the list of PR is probably enough for us.

**Review by BarnyShergold — APPROVED** [review]
This is a great addition and resource. The second reviewer may have additional suggestions

File: src/community/resources/contribution-day-organizer-handbook.md (hunk 3 above)
Comment (sivaschenko) [reply src/community/resources/contribution-day-organizer-handbook.md]: I believe GitHub usernames are often required to provide additional access that may be needed during the event

Comment (gabrieldagama) [reply src/community/resources/contribution-day-organizer-handbook.md]: The topic on this one is related to `Tracking Contributions`, as currently, we have Bitergia to analyze the contributions I'm not sure if need that information. At the `Registration Page` it is still recommended for the organizer to collect GitHub username.

**Review by sivaschenko — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8769`.

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8769`.

**Comment by dshevtsov** [comment]
@dobooth, the whatsnew is missing in this PR.

**Pull request merged**